test/unit/scratchOrgCreateTest.ts

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
I would have expected more tests for org create. Are there plans to increase unit testing for this?